docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -154,10 +154,15 @@
         path: /etc/platform/.unlock_ready
       register: is_unlock_ready
 
+    - name: Check if host is provisioned
+      shell: |
+        source /etc/platform/openrc;
+        system host-show "{{ get_host_name.stdout}}" | awk '$2 ~ /^invprovision/ {print $4}'
+      register: get_host_invprovision
```

Comment:
This register is not used in the playbook

Reply:
As discussed, i will close this PR. we will go with -
https://github.com/Wind-River/cloud-platform-deployment-manager/pull/390